styles/branchingQuestion.css

```diff
@@ -59,7 +59,7 @@
 }
 
 .h5p-branching-scenario .h5p-branching-question button:not(.h5p-branching-question-alternative) {
-  margin-top: 1em;
+  margin: 1em 0.188em 0.188em 0.188em;
```

Comment:
Here you can use the shorthand property for setting margin, meaning you can skip the last value. 
`margin: 1em 0.188em 0.188em`
=> 
top: 1em
left and right: 0.188em
bottom: 0.188em